server/gc/src/main/java/org/apache/accumulo/gc/GCRun.java

```diff
@@ -462,4 +479,28 @@ public long getErrorsStat() {
   public long getCandidatesStat() {
     return candidates;
   }
+
+  @Override
+  public boolean isRootTable() {
+    return level == DataLevel.ROOT;
+  }
+
+  @Override
+  public boolean isMetadataTable() {
+    return level == DataLevel.METADATA;
+  }
+
+  @Override
+  public Set<TableId> getCandidateTableIDs() {
```

Comment:
Using level in the name seems better to me.

```suggestion
  /**
   * @return the tables id for the current data level
   */
  public Set<TableId> getLevelTableIDs() {
```

Reply:
Looking back at the 1.10 PR this method is called getCandidateTableIDs... so would probably want to keep the same method name in this branch.